localstack/testing/snapshots/transformer_utility.py

```diff
@@ -165,18 +165,36 @@ def sqs_api():
         """
         return [
             TransformerUtility.key_value("ReceiptHandle"),
-            TransformerUtility.key_value("SenderId"),
+            # account ID
+            TransformerUtility.key_value(
+                "SenderId", value_replacement="111111111111", reference_replacement=False
+            ),
```

Comment:
I had a similar issue in #6603. It seems related to the the fact that the `SenderId` of an SQS message can sometimes be the root account ID, and sometimes the access key id, see. https://stackoverflow.com/a/28603793/804840
This then messes up the comparisons.

Any idea what a good pattern is for this @steffyP @dominikschubert ?

Reply:
@thrau: Interesting :) I think a key-value replacement should be fine in that case. We could also include a regex-check to make sure it matches one of the two valid patterns. but seems like a overhead in that case.

Only thing that feels a bit hacky here: if we replace it with 1s, one could assume that the region-regex-replacement was used here - which is not the case. @bentsku is there a particular reason why you have chosen this value?